README.md

```diff
@@ -488,26 +527,33 @@ interface IAndroidApplicationInformation {
 
 Sample usage:
 ```JavaScript
-require("mobile-cli-lib").devicesService.getApplicationsAvailableForDebugging("4df18f307d8a8f1b")
-	.then(function(applicationsInfo) {
-		applicationsInfo.forEach(function(applicationInfo) {
-			console.log(applicationInfo);
+Promise.all(require("mobile-cli-lib").devicesService.getDebuggableApps(["4df18f307d8a8f1b"]))
```

Comment:
add `JJY5KBTW75TCHQUK` to the array passed to `getDebuggableApps` as the sample result contains results for this identifier as well.